frontend-implicaction/src/app/auth/components/signup/signup.component.ts

```diff
@@ -67,7 +69,10 @@ export class SignupComponent implements OnInit {
       .signup(this.signupRequestPayload)
       .pipe(finalize(() => this.isLoading = false))
       .subscribe(
-        () => this.router.navigate(['/auth/login'], {queryParams: {registered: 'true'}}),
+        () => {
+          this.router.navigate(['/auth/login'], {queryParams: {registered: 'true'}})
```

Comment:
```suggestion
          this.router.navigate(['/auth/login'])
```
plus besoin de préciser le paramètre grâce au alert-service